components/Slider/src/index.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
+	import { createEventDispatcher } from "svelte";
+
+	export let value: number = 0;
+	export let min: number = 0;
+	export let max: number = 100;
+	export let step: number = 1;
+	export let disabled: boolean = false;
+	export let attrs = {};
+	export let cls = '';
+
+	// current value
+	let isDragging: boolean = false;
+	let startX: number = 0;
+	let startPosition: number;
+	let newPosition: number;
+
+	$:percentage = `${(value - min) / (max - min) * 100}%`;
+
+	// element
+	let sliderRunwayRef: null | HTMLElement = null;
+	// updateValue
+	const dispatch = createEventDispatcher();
+	const handleMouseEnter = () => {
```

Comment:
Seems like the component isn't finished yet?

Reply:
You're absolutely right. The Slider conponent isn't fully finished yet. It's currently a basic version.

Reply:
You're absolutely right. The component isn't fully finished yet. It's currently a basic version